lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,29 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # The class we work with
+  class Car
+    attr_accessor :year, :color, :model, :current_speed
+
+    def initialize(args)
```

Comment:
If I add default data it fails one of the tests - "expects hash of attributes". In other words, it stops raising ArgumentError.